Library/Formula/mpd.rb

```diff
@@ -116,4 +119,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    pid = fork { exec "#{bin}/mpd --stdout --no-daemon --no-config" }
+    sleep 2
+    Process.kill("SIGINT", pid)
+    Process.wait(pid)
+  end
```

Comment:
Alright, I don't know. We could go ugly and obvious as in:

```ruby
  test do
    icu4c = Formula["icu4c"].opt_lib/"libicui18n.55.dylib"
    assert_match /#{icu4c}/, shell_output("otool -L #{bin}/mpd")
  end
```

But everyone involved, me, you, all of the maintainers aside you, etc are going to hate the test. It works, but I don't really want to do it. Going to cc @jacknagel in case he knows of any Ruby magic on this.